docs/application/tizen-studio/release-notes/2-4-release-notes.md

```diff
@@ -0,0 +1,86 @@
+# Tizen Studio 2.4 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Device Manager
+	-   Enable Device Manager Context UI to be extended by plugins
+-   RT-IDE
+	-   RT-IDE GCC Toolchain update to 6.3.1
+-   SmartThings
+	-   Make setupId mandatory field and map it to device onboarding id
+	-   Updated ramls for smart things resources
+-   Package Manager
+	-   Remove *.po files getting packaged into tpk
+
+### Changed Features
+-	RT-IDE
+	-   Removed local template for creating project. Project can be created using latest RT code from github.
+
+### Fixed Bugs
+-   Device Manager
```

Comment:
Device Manager 
•	Fixed the crash that occurred while filtering devices in JAVA 9.
•	Increased the clickable area for Add new tab button to cover the entire button area.
•	Added support for empty filter category in logs.
•	Fixed permit to install the failure for secure TV profile.